ipalib/rpc.py

```diff
@@ -226,7 +226,7 @@ def xml_unwrap(value, encoding='UTF-8'):
     if type(value) in (list, tuple):
         return tuple(xml_unwrap(v, encoding) for v in value)
     if type(value) is dict:
-        if '__dns_name__' in value:
+        if '__dns_name__' in value:  # lgtm [py/member-test-non-container]
```

Comment:
This kind of type case must be very common.  I wonder... what happens if you change
the conditional
```python
if type(value) is dict:
```
to
```python
if isinstance(value, collections.Mapping):
```
does *LGTM* recognise the latter more idiomatic/robust construction and stop complaining?

Reply:
I'll try that now